Modules/SubModule_Structural/xASL_wrp_GetVolumetrics.m

```diff
@@ -94,21 +94,29 @@ function xASL_wrp_GetVolumetrics(x)
     xASL_adm_DeleteFileList(x.D.TissueVolumeDir, '^LST_tlv_.*\.(csv|tsv)$', [], [0 Inf]);
     xASL_adm_DeleteFileList(x.D.TissueVolumeDir, ['WMH_LST_(LGA|LPA)_' x.P.SubjectID '.(csv|tsv)'], [], [0 Inf]);
     
-    % run the LST lesion volume calculator
+    % Run the LST lesion volume calculator
+	% Change directory to the SUBJECT DIRECTORY as the count function outputs to the current directory
```

Comment:
"count function" is not clear, it took me some time to understand that you meant `ps_LST_tlv`, (am I right?) Please be explicit ;)

Reply:
sure - you can update the comment...

Reply:
This should be the new comment:

	% -> Here, we change the current directory to the SUBJECT DIRECTORY as ps_LST_tlv outputs to the current directory

But if I add a new commit, it becomes difficult for you to undo your comment...